### ISSUE TITLE
Strange chequered artifacting around transparencies in Fallout 3 / New Vegas

### ISSUE THREAD

**Issue by brad86m** [doitsujin/dxvk#3380]
Not sure what the cause is, but I have changed settings within the New Vegas launcher, and in DXVK config, but can't seem to fix this.

### System information
- GPU: RTX 3090
- Driver: 526.86
- Wine version:  n/a
- Windows version: 11
- DXVK version: 2.1

![NO DXVK](https://user-images.githubusercontent.com/84683672/234332577-256361f7-ebaa-4f4b-bdf8-061de9ae9926.png)
![With DXVK](https://user-images.githubusercontent.com/84683672/234332601-ee1220a2-9f1b-4712-ab9c-af38d226aa6c.png)

**Comment by K0bin** [comment]
Looks like the game decided to use dithering instead of blending for *some reason*.

**Comment by brad86m** [comment]
> 

Absolutely will. Not sure how to apitrace, though.

**Comment by brad86m** [comment]
I have just found a post on Nexus mods mentioning that this is a bug since version 1.10.1

**Comment by Blisto91** [comment]
If you can send a savegame of the location i can look into making a trace. If i reproduce on AMD.

**Comment by K0bin** [comment]
> I have just found a post on Nexus mods mentioning that this is a bug since version 1.10.1

In that case, someone could bisect it to find out which commit made the game do it.

**Comment by Blisto91** [comment]
> Isn't this the same as #3222?

Yes i think so.

From what i can tell this also isn't a regression in 1.10.1 but has looked like this ever since d9vk 0.30, which actually is close to native than before.


Apologize for the full desktop screenshots, Windows screenshots and dxvk doesn't work well for me.
Also apologize they are taken a bit far away from the tree but the issue disappears a bit when you get closer. Viewing at full size or a bit of zoom you should be able to spot it.
Lastly i don't know why the ground foliage moves around.
<details>
  <summary>Screenshots</summary>
Native game
![1_native](https://user-images.githubusercontent.com/47954800/234382446-7dd21cb1-3794-4489-a9f2-0d49552fa60e.png)

d9vk 0.22
![2_d9vk-0 22](https://user-images.githubusercontent.com/47954800/234382524-5a95770a-7e72-4bc5-8e7f-35fe4737bac9.png)

d9vk 0.30
![3_d9vk-0 30](https://user-images.githubusercontent.com/47954800/234382606-e2cf5220-9acd-474b-8876-09beb626dc1d.png)

dxvk 2.1
![4_dxvk-2 1](https://user-images.githubusercontent.com/47954800/234382664-6bc00138-1e3c-47cb-a413-5fba39370865.png)
</details>

**Comment by brad86m** [comment]
Thanks for looking deeper into it, guys. I wonder what it could be.

**Comment by DadSchoorse** [comment]
It's almost certainly alpha to coverage dithering. On radv it can be disabled with `RADV_DEBUG=noatocdithering` to get native d3d9 behavior (no dithering). To make it work by default and on more vendors we would need a vulkan extension, but that's very low on the priority list given the minor differences in visuals.

**Comment by JJones780** [comment]
Is it only with tree branches?  I see this checkerboard texture affect with tree branches (only?)  in IL2-Sturmovik as well. I have bigger issues when smoke appears - which has alpha but does not exhibit this dithered artifact so probably not related?

**Comment by neatodev** [comment]
There is no way to force dxvk to change behavior with the config file?

**Comment by K0bin** [comment]
Nope, there isn't. It would require changes to the Vulkan driver.

**Comment by dlpnd** [comment]
Tested with latest build, seems to be much better 

https://cdn.discordapp.com/attachments/825404977965957200/1120948982567411814/image.png

**Comment by Blisto91** [comment]
Checking in game it look to be exactly the same as before to me. It look like you are standing too close in your screenshot.

<details>
  <summary>Screenshot</summary>
  
![Screenshot 2023-06-21 094937](https://github.com/doitsujin/dxvk/assets/47954800/0cf7acea-aa0f-43e2-9483-3e7f3c092b05)
</details>

**Comment by dlpnd** [comment]
![image](https://github.com/doitsujin/dxvk/assets/11916166/bf15a524-7f18-482f-aa19-a436ba06939d)
I tried standing in the similar area, there may be some artifacting, but way less. 

I've got 6700XT and in Windows 11 if that makes a difference
